SPECS/gnutls/gnutls.spec

```diff
@@ -1,20 +1,19 @@
 Summary:        The GnuTLS Transport Layer Security Library
 Name:           gnutls
-Version:        3.7.7
-Release:        3%{?dist}
+Version:        3.8.3
```

Comment:
Why the version bump? We try to avoid minor-version number changes within one version of Azure Linux. Is a patch for the current version feasible?